EFCodeFirst/EFCodeFirst/Program.cs

```diff
@@ -0,0 +1,261 @@
+            context.Database.ExecuteSqlCommand("DELETE FROM dbo.Customers");
+        }
+        static void prepareDB(Context context)
+        {
+            clearDB(context);
+
+            prepareCustomers(context);
+            prepareSuppliers(context);
+            prepareProducts(context);
+            prepareOrders(context);
+        }
+        static void prepareCustomers(Context context)
+        {
+            var customer1 = new Customer
+            {
+                cutomerName = "Petya",
+                customerEmail = "petya@email.com"
+            };
+            var customer2 = new Customer
+            {
+                cutomerName = "Vasya",
+                customerEmail = "vasya@email.com"
+            };
+            var customer3 = new Customer
+            {
+                cutomerName = "Kolya",
+                customerEmail = "kolya@email.com"
+            };
+
+            context.Customers.Add(customer1);
+            context.Customers.Add(customer2);
+            context.Customers.Add(customer3);
+
+            context.SaveChanges();
+        }
+        static void prepareSuppliers(Context context)
+        {
+            var supplier1 = new Supplier
+            {
+                supplierName = "Rozetka",
+                supplierPhone = "+38(098)-988-88-88",
+                supplierEmail = "supplier@rozetka.ua"
+            };
+            var supplier2 = new Supplier
+            {
+                supplierName = "Citrus",
+                supplierPhone = "+38(099)-999-09-09",
+                supplierEmail = "supplier@citrus.ua"
+            };
+
+            context.Suppliers.Add(supplier1);
+            context.Suppliers.Add(supplier2);
+
+            context.SaveChanges();
+        }
+        static void prepareProducts(Context context)
+        {
+            var suppliers = context.Suppliers.Take(2).ToArray();
+
+            var product1 = new Product
+            {
+                productName = "IPhone 18 512TB",
+                productPrice = 5000,
+                productType = "IPhone",
+                supplier = suppliers[0]
+            };
+            var product2 = new Product
+            {
+                productName = "IPhone 18 PRO 1024TB",
+                productPrice = 7000,
+                productType = "IPhone",
+                supplier = suppliers[0]
+            };
+            var product3 = new Product
+            {
+                productName = "Samsung Galaxy S50 512TB",
+                productPrice = 5000,
+                productType = "Samsung Galaxy series",
+                supplier = suppliers[1]
+            };
+
+            context.Products.Add(product1);
+            context.Products.Add(product2);
+            context.Products.Add(product3);
+
+            context.SaveChanges();
+        }
+        static void prepareOrders(Context context)
+        {
+            var products = context.Products.Take(2).ToArray();
```

Comment:
same here

Reply:
Done in cd93496ab08d07c56a756b70c234cd198533c6a4